uwblueprint/starter-code-v2
# Create starter-code-v2 release

## Notion ticket link

[Create starter code release](https://www.notion.so/uwblueprintexecs/Create-starter-code-release-fc33bcd1ce824309a4a290215ff7d574)



## Implementation description
* Followed [release instructions](https://github.com/uwblueprint/starter-code-v2#creating-a-release)
* Release includes all `main` commits from de3c238c29f6548d49b6d9a4bab8610843d374fa onwards (inclusive)



## Steps to test
1. Checkout https://github.com/uwblueprint/starter-code-cli/pull/21
2. For each of the 12 stack combinations: generate starter-code using the CLI, make a request to every endpoint, ensure the frontend compiles
```bash
# run starter-code-cli locally
# when prompted for a directory, give the path to the directory 1 level above your local starter-code-v2
yarn dev --testing

# after testing, reset starter-code-v2
git reset --hard HEAD
```
3. If the stack requires migrations, follow instructions in #48 to test migrations
4. If using a REST API: check that `localhost:5000/api-docs` returns Swagger UI and that the correct endpoints are documented
5. If using GraphQL, check that `localhost:5000/graphql` returns the GraphQL playground



## What should reviewers focus on?
* Take a look at any file that you've touched recently and check to see if anything's out of place


## Checklist
- [x] N/A - I have run the appropriate linter(s)
